extension-manifest-v3/src/utils/custom-filters.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+const KEY_DNR_RULES = 'custom-filters-dnr';
+
+export async function getDNRRules() {
+  return (await chrome.storage.local.get([KEY_DNR_RULES]))[KEY_DNR_RULES] || [];
+}
+
+export async function setDNRRules(filters = []) {
+  await chrome.storage.local.set({
```

Comment:
I'm not convinced if this storage is needed. If we don't want to show the DNR conversion output (after the reload) in UI we can remove it.